app/config/sculpin_kernel.yml

```diff
@@ -78,7 +78,7 @@ sculpin_content_types:
         type: path
         path: docs/guides/essentialdevelopertraining
```

Comment:
I suggest path: `docs/guides/edt` 

Simpler and shorter links makes them easier to recall. This is too long imo

Reply:
Agreed and updated.